src/Select/Select.js

```diff
@@ -0,0 +1,207 @@
+// @flow
+import * as React from "react";
+import defaultTokens from "@kiwicom/orbit-design-tokens";
+import styled from "styled-components";
+
+import FormLabel from "../FormLabel/FormLabel";
+import ChevronDown from "../icons/ChevronDown";
+import FormFeedback, { TYPE_OPTIONS } from "../FormFeedback/FormFeedback";
+
+export const SIZE_OPTIONS = {
+  SMALL: "small",
+  NORMAL: "normal",
+};
+
+type Option = {
+  value: string | number,
+  label?: string,
+  disabled?: boolean,
+};
+
+type Props = {
+  size: $Values<typeof SIZE_OPTIONS>,
+  label?: string,
+  placeholder?: string,
+  value: string,
+  disabled?: boolean,
```

Comment:
When select is disabled, styles are not changed.

Reply:
Good catch 👏